OPHD/States/Planet.cpp

```diff
@@ -69,7 +69,7 @@ void Planet::update()
 	}
 
 	auto& renderer = NAS2D::Utility<NAS2D::Renderer>::get();
-	const auto spriteFrameOffset = NAS2D::Point{mTick % 8 * PlanetSize.x, ((mTick % 64) / 8) * PlanetSize.y};
+	const auto spriteFrameOffset = NAS2D::Point{mTick % 16 * PlanetSize.x, ((mTick % 256) / 16) * PlanetSize.y};
```

Comment:
Should probably be updating the submodule in the same commit as this, since things break when they are updated separately. Not that it really matters that much for back to back commits.